lib/index.tsx

```diff
@@ -8,9 +8,10 @@ import {
 type FlagrContextType = {
   evaluate?: <T>(flag: string, callbacks: FlagCallbacks<T>) => T;
   match?: (flag: string, matchVariant?: string) => boolean;
+  evaluating: boolean;
```

Comment:
**question/suggestion** I think we use `loading` as the term for other hooks, this seems to be along the same lines, so thoughts on renaming?

Reply:
@chintogtokh updated to `loaded`